packages/devtools_server/lib/src/memory_profile.dart

```diff
@@ -58,68 +58,68 @@ class MemoryProfile {
 
   void handleServiceEvent(Event e) {
     if (e.kind == EventKind.kServiceRegistered) {
-      final serviceName = e.service;
+      final serviceName = e.service!;
       _registeredMethodsForService
-          .putIfAbsent(serviceName, () => [])
-          .add(e.method);
+          .putIfAbsent(serviceName, () => [])!
+          .add(e.method!);
     }
 
     if (e.kind == EventKind.kServiceUnregistered) {
-      final serviceName = e.service;
+      final serviceName = e.service!;
       _registeredMethodsForService.remove(serviceName);
     }
   }
 
-  IsolateRef _selectedIsolate;
+  late IsolateRef _selectedIsolate;
 
-  Future<Response> getAdbMemoryInfo() async {
+  Future<Response?> getAdbMemoryInfo() async {
     return await callService(
       registrations.flutterMemory.service,
       isolateId: _selectedIsolate.id,
     );
   }
 
   /// Call a service that is registered by exactly one client.
-  Future<Response> callService(
+  Future<Response?> callService(
     String name, {
-    String isolateId,
-    Map args,
+    String? isolateId,
+    Map<String, dynamic>? args,
   }) async {
     final registered = _registeredMethodsForService[name] ?? const [];
     if (registered.isEmpty) {
       throw Exception('There are no registered methods for service "$name"');
     }
-    return service.callMethod(
+    return service!.callMethod(
       registered.first,
       isolateId: isolateId,
       args: args,
     );
   }
 
-  Map<String, List<String>> get registeredMethodsForService =>
+  Map<String, List<String>?> get registeredMethodsForService =>
```

Comment:
seems odd that this is `List<String>?` instead of `List<String>`

Reply:
fixed